awsbatch-cli/setup.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from setuptools import find_packages, setup
+
+
+def readme():
+    """Read the README file and use it as long description."""
+    with open(os.path.join(os.path.dirname(__file__), "README")) as f:
+        return f.read()
+
+
+VERSION = "2.10.3"
```

Comment:
probably we don't want to keep this version locked with pcluster one. we might start from version 1.0.0?what do you think?

Reply:
It's a good point to discuss, I had the same thought when creating the patch

Reply:
Changed to 1.0.0